src/api/census3/census.ts

```diff
@@ -89,19 +127,39 @@ export abstract class Census3CensusAPI extends Census3API {
       .catch(this.isApiError);
   }
 
+  /**
+   * Returns the information of the census queue
+   *
+   * @param {string} url API endpoint URL
+   * @param {string} id The identifier of the census queue
+   * @returns {Promise<ICensus3CensusQueueResponse>}
+   */
+  public static queue(url: string, id: string): Promise<ICensus3CensusQueueResponse> {
+    return axios
+      .get<ICensus3CensusQueueResponse>(url + Census3CensusAPIMethods.QUEUE.replace('{id}', id))
+      .then((response) => response.data)
+      .catch(this.isApiError);
+  }
+
   /**
    * Requests the creation of a new census with the strategy provided for the blockNumber.
    *
    * @param {string} url API endpoint URL
    * @param {number} strategyId The strategy identifier
+   * @param {boolean} anonymous If the census has to be anonymous
    * @param {number} blockNumber The number of the block
    * @returns {Promise<ICensus3CensusCreateResponse>} promised ICensus3CensusCreateResponse
    */
-  public static create(url: string, strategyId: number, blockNumber?: number): Promise<ICensus3CensusCreateResponse> {
+  public static create(
+    url: string,
+    strategyId: number,
+    anonymous: boolean = false,
+    blockNumber?: number
+  ): Promise<ICensus3CensusCreateResponse> {
     return axios
       .post<ICensus3CensusCreateResponse>(
         url + Census3CensusAPIMethods.CREATE,
-        JSON.stringify({ strategyId, blockNumber })
+        JSON.stringify({ strategyId, blockNumber, anonymous })
```

Comment:
Why use JSON.stringify? Doesn't axios.post already do it internally? 🤔

Reply:
New issue #248